### PR TITLE
Only append parameter when it was provided in the first place

### PR DESCRIPTION
**Pull request by maartenba**
This fixes the scenario described at https://social.msdn.microsoft.com/Forums/en-US/6cc0ef87-80ec-4331-8b41-efb22d8f1c60/odata-provider-with-wcf-40-paging-operations-with-null-parameters?forum=adodotnetdataservices.

We ran into it on building NuGet as well:

* https://github.com/NuGet/NuGetGallery/issues/1502
* https://github.com/NuGet/NuGetGallery/issues/1886
* https://github.com/NuGet/NuGetGallery/issues/1259

Long story short: when an OData service operation `Foo(string bar, string baz)` is called, but `baz` is not provided (`null`), the `GetNextPageQueryParametersForRootContainer()` method on `Serializer` was trying to escape a missing query string parameter, resulting in a `NullReferenceException`.

This pull request fixes this exception by not trying to escape non-existant query string parameters.

### CODE REVIEW

**Comment by congysu** [comment]
We might need to take a look at v3 spec.

**Comment by maartenba** [comment]
Awesome, thanks! Do you have an estimate when a new NuGet package will be ready?

**Comment by congysu** [comment]
@maartenba: We released [5.6.5 beta](http://www.nuget.org/packages/Microsoft.Data.Services/5.6.5-beta), can you see if it works?  Thanks.

**Comment by maartenba** [comment]
Works like a charm!
